portable/NetworkInterface/ThirdParty/MSP432/NetworkInterface.c

```diff
@@ -247,10 +257,14 @@ bool setupEMAC()
 {
     uint32_t ul32Loop;
     bool rv;
+    bool interruptsMasked;
```

Comment:
maintainability: it's better to initialize these inline in case a future maintainer inserts code between declaration and assignment, e.g.:

```
bool interruptsMasked = false;
```

Reply:
I agree, sometimes I think in code prior to C99.  This has now been updated.